py/desispec/ccdcalib.py

```diff
@@ -458,6 +505,13 @@ def compute_nightly_bias(night, cameras, outdir=None, nzeros=25, minzeros=20,
 
     nfail = 0
     for camera in cameras[rank::size]:
+        if camera in skipped_cams:
+            log.error(f'execution was skipped for camera {camera} due to lack of usable zeros')
+            nfail+=1
+            continue
+
+        expids=expdict[camera]
+        rawfiles=rawfiles_dict[camera]
```

Comment:
Replace with logic from line 488 and use the `expids` you just took from the `expdict`.